src/types/Account/_initializeAccount.js

```diff
@@ -1,11 +1,7 @@
-const _ = require('lodash');
 const logger = require('../../logger');
-const TransactionSyncStreamWorker = require('../../plugins/Workers/TransactionSyncStreamWorker/TransactionSyncStreamWorker');
-const ChainPlugin = require('../../plugins/Plugins/ChainPlugin');
-const IdentitySyncWorker = require('../../plugins/Workers/IdentitySyncWorker');
 const EVENTS = require('../../EVENTS');
 const { WALLET_TYPES } = require('../../CONSTANTS');
-
+const preparePlugins = require('./_preparePlugins');
```

Comment:
Don't use underscores please :)

Reply:
How would you suggest I can differentiate them from public function (this is purely internals created in their own file for sake of better testing)